incubator/webpagetest-server/templates/secrets-user-details.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.ec2Locations.enabled }}
+apiVersion: v1
+kind: Secret
```

Comment:
I'm glad you're passing these credentials as a secret. One pattern we would like to see is allowing someone to set the secret outside the chart if they want. If they want the chart to create it they can have that as an option. For example, see the wordpress chart with tls secrets... https://github.com/kubernetes/charts/blob/master/stable/wordpress/values.yaml#L145

Could you look at something like this?